crates/rpc/rpc-eth-types/src/receipt.rs

```diff
@@ -118,8 +116,7 @@ impl ReceiptBuilder {
     }
```

Comment:
at second thought, this method should be removed as part of this pr, because those fields won't be contained in the final receipt built